### ISSUE TITLE
Potential conflict when expanding causal relation with coreference chain

### ISSUE THREAD

**Issue by yulinchen99** [tommasoc80/EventStoryLine#10]
https://github.com/tommasoc80/EventStoryLine/blob/f3c9aad79bb5acd1be3c5752c3493e0d4fa62013/create_gold_document.py#L141

Hi, I found the processing script here does not consider conflicted relation inference. It is possible for one pair of mentions to have "PRECONDITION" and "FALLING_ACTION" relations inferred at the same time. The conflicted pairs would better be deleted.

**Comment by cyl628** [comment]
> Thanks for pointing this out. Did you find any actual inconsistencies?

Yes. For topic 1, some inconsistency examples are
(the format is `file (source, target) relation1 relation2`)
```
1_18ecbplus.xml ('45', '150') PRECONDITION FALLING_ACTION
1_18ecbplus.xml ('92_93', '150') PRECONDITION FALLING_ACTION
1_21ecbplus.xml ('32_33', '53') FALLING_ACTION PRECONDITION
1_21ecbplus.xml ('47_48', '53') FALLING_ACTION PRECONDITION
```

**Comment by cyl628** [comment]
> Thanks for pointing this out. Did you find any actual inconsistencies?

Yes. For topic 1, some inconsistency examples are
(the format is `file (source, target) relation1 relation2`)
```
1_18ecbplus.xml ('45', '150') PRECONDITION FALLING_ACTION
1_18ecbplus.xml ('92_93', '150') PRECONDITION FALLING_ACTION
1_21ecbplus.xml ('32_33', '53') FALLING_ACTION PRECONDITION
1_21ecbplus.xml ('47_48', '53') FALLING_ACTION PRECONDITION
```

**Comment by tommasoc80** [comment]
Right. I don't have time to take care of that, but if you are willing to push code and revised annotation, I will make them available and acknowledge your contribution to the project.
